src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/MethodeLinksBodyProcessor.java

```diff
@@ -71,51 +78,40 @@ public MethodeLinksBodyProcessor(ResilientClient documentStoreApiClient, URI uri
     @Override
     public String process(String body, BodyProcessingContext bodyProcessingContext) throws BodyProcessingException {
         if (body != null && !body.trim().isEmpty()) {
```

Comment:
And why not throw there directly `BodyProcessingException` instead of `IllegalStateException`.